tests/tdd-tutorial/solver_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rspec'
+require_relative 'solver'
+
+describe Solver do
+  describe '#factorial' do
+    it 'returns 120' do
+      five = Solver.new
+      expect(five.factorial(5)).to eq(120)
+    end
+
+    it 'returns "No factorial for negative integers"' do
+      negative = Solver.new
+      expect(negative.factorial(-1)).to eq('No factorial for negative integers')
+    end
```

Comment:
- [ ] Please from the project requirement, if a negative integer is given it should raise an exception so your need to raise an exception not a to equal to a string :pray: 
like :point_down: 
`expect { resolver.factorial(-5) }.to raise_exception`

Reply:
Fixed!